element/index.js

```diff
@@ -28,3 +28,8 @@ export function createElement( type, props, ...children ) {
 export function render( element, target ) {
 	reactRender( element, target );
 }
+
+/**
+ * A base class to create WordPress Components (Refs, state and lifecycle hooks)
+ */
+export const Component = ReactComponent;
```

Comment:
Prefixed imports bothering me a bit. We could instead do...

```js
import { Component } from 'react';

// ...
export { Component };
```

What do you think?